app/forms/withdrawal/future_interest_form.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Withdrawal
+  class FutureInterestForm < TraineeForm
+    FIELDS = %i[future_interest].freeze
+
+    attr_accessor(*FIELDS)
+
+    validates :future_interest, presence: true, inclusion: { in: %w[yes no unknown] }
+
+  private
+
+    def form_store_key
+      :withdrawal_future_interest
+    end
```

Comment:
I think this is automatically defined in the base class, does it work ok if you get rid of it?

Reply:
Right, because these attributes are associated with a child record, I thought it made sense to give them a prefix to be able to deal with them separately from the trainee attributes on save.